### DISCUSSION TITLE
Can't change the download location on Fedora 38 (Flatpak)

### DISCUSSION THREAD

**Discussion by soulis-1256** [KRTirtho/spotube#693] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

"Download location" in the "Download" section of Settings isn't responsive when clicking it, therefore I am unable to change the download directory to other than the default.

### Expected Behavior

The "Download location" component should open the file manager to select a directory.

### Steps to reproduce

1. Open the app (flatpak, fedora workstation 38, version 3.1.1)
2. Go to Settings
3. Go to "Download location" in the Download section
4. Click either the directory icon, or anywhere else in the component

### Operating System

Linux

### Spotube version

v3.1.1

### Installation source

Flathub (Flatpak)

### Additional information

![image](https://github.com/KRTirtho/spotube/assets/118274635/48f65929-2994-4ea3-b741-1810ecdd8c09)
You can clearly see what I'm talking about.

**Comment by KRTirtho** [comment]
Changing location isn't supported on Flatpak due to lack of filesystem permissions

**Reply by soulis-1256** [reply]
I have to point out 2 things:
1. I tried changing filesystem perms using Flatseal, but nothing changed.
2. Even if it's currently impossible to achieve this, shouldn't you disable the component for flatpak, or just add a general warning around that area?

**Reply by dc-me** [reply]
I try to change it to user's music folder, also not working, used flatseal also. I think default to user's musci folder is perfered.
